erpnext/patches.txt

```diff
@@ -330,5 +330,8 @@ execute:frappe.db.sql("update `tabTimesheet` ts, `tabEmployee` emp set ts.employ
 erpnext.patches.v7_1.update_lead_source
 erpnext.patches.v7_1.fix_link_for_customer_from_lead
 erpnext.patches.v7_0.update_mode_of_payment_type
+
+execute:frappe.reload_doctype('Employee')
+execute:frappe.db.sql("update `tabEmployee` set prefered_contact_email = IFNULL(prefered_contact_email,'') ")
```

Comment:
the reason for it is due to when a select field is add and has a frappe.ui.form.on hook it changes the value from null to empty causing the event trigger to trigger before the page has loaded properly resulting in the child field updating with the same value as the previous open'd employee (only occurs when you change from one employee to the other with refreshing)

if we set to company email n patch so has correct data then you cant select a blank option for opt out (for the people without email)